src/file-uploader.scss

```diff
@@ -25,6 +25,22 @@ $fd-file-uploader-padding: 0.25rem;
       margin-left: $fd-file-uploader-padding;
       margin-right: 0;
     }
+
+    &.is-error {
+      background-color: var(--sapField_InvalidBackground);
+      border-color: var(--sapField_InvalidColor);
+    }
+
+    &.is-warning,
+    &.is-alert {
+      background-color: var(--sapField_WarningBackground);
+      border-color: var(--sapField_WarningColor);
+    }
+
+    &.is-information {
+      background-color: var(--sapField_InformationBackground);
+      border-color: var(--sapField_InformationColor);
+    }
```

Comment:
It's missing the is-success state

Reply:
`.fd-file-uploader__input` is always used with `fd-input` and `fd-input` already has `is-warning`, `is-success`, etc... Why do you bring them again?

Reply:
i have tried to use ``is-error`, `is-success' with fd-input but styles are not applied. so it required to get them here. 

>

Reply:
<img width="942" alt="Screen Shot 2021-04-20 at 10 06 36 AM" src="https://user-images.githubusercontent.com/39598672/115410014-477b7c80-a1c0-11eb-95d2-db6576fac2ea.png">

Hi @manu-kr the modifiers are not applied because the input is `readonly`. You don't have to introduce states in the File Uploader but check the states of Input and fix the problem there.

Reply:
As i discussed with you i have removed readonly attribute for input field. and make some additional required changes in css. and it requires some logical changes in ngx library after this PR merge i can make those changes.